CHANGELOG.rst

```diff
@@ -20,11 +20,15 @@ This document describes changes between each past release.
 - Server now returns 415 error response if client cannot accept JSON response (#461, mozilla-services/cliquet#667)
 - Server now returns 415 error response if client does not send JSON request (#461, mozilla-services/cliquet#667)
 - Add the ``__lbheartbeat__`` endpoint, for load balancer membership test.
+- Expose the experimental json-schema feature in the public settings on the hello_page (fixes #270)
```

Comment:
As @almet said, and if I understood correctly, this might not be mentioned as a protocol change but as a new feature!